src/constants/approval-routes.js

```diff
@@ -0,0 +1,31 @@
+const routes = {
```

Comment:
This is maybe a silly suggestion, but can we alphabetize these, or does the order matter? I thought the `allrequests` and `allrequest` sections were duplicates at first until I realized one was plural.

Reply:
They can be alphabetized.